packages/plugin-activedirectory-api/src/models/definitions/actives.ts

```
@@ -8,6 +8,7 @@ export interface IConfig {
   code: string;
   isLocalUser: string;
   userDN: string;
+  baseDN: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)